backend/src/main/java/de/amos/apachepulsarui/dto/TopicDto.java

```diff
@@ -33,6 +35,8 @@ public class TopicDto {
 
 	private long consumedMessages;
 
+	private MessagesDto messagesDto;
```

Comment:
You need to make sure that the List is always initialized otherwise the json mapping will fail (the tests fail with a 500 error which was very confusing for me, before I figured out that this was the problem).